### PR TITLE
Feature/example notebooks for cape core

### PR DESCRIPTION
**Pull request by kjam**
This pull request shares some example Jupyter Notebook and policy files for the Cape Core release.

### CODE REVIEW

File: examples/notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`cape-privacy` gives you the ability to apply several masking techniques (transformations) such as tokenization, perturbation, rounding, etc., in order to obfuscate personal information contained in your dataset. You can find out more by visiting [our documentation](https://docs.capeprivacy.com/libraries/cape-python/).
+
+## Notebook Overview
+
+There are several posts related to these Jupyter notebooks to assist you in evaluating the code and privacy considerations. The related policy files can be found in the `policy` folder in this `examples` folder. You will find links to the datasets in the individual notebooks. Note that some datasets are reused, so you might need to download only once.
+
+### Cape Core: Introduction to Collaborative Privacy and Security Policy
+
+This [overview of Cape Core software]() walks you through the use of Cape Core alongside Cape Privacy. It is a great way to get started with managing your transformations in a responsible way. Cape Core allows you to store policies centrally and coordinate them with your data science and machine learning peers. There are two notebooks related to this blog post:
```
Comment (justin1121) [inline examples/notebooks/README.md:11]: Missing link at:

> over of Cape Core software

Comment (kjam) [reply examples/notebooks/README.md:11]: ah nice - thanks for the catch!!

**Comment by kjam** [comment]
@yanndupis let me know if these notebooks look good to you -> then please feel free to squash and merge - thank you! :pizza:

**Comment by yanndupis** [comment]
@kjam it looks great, great work 💪 !! as you suggested, let's move these examples to a separate repo next week :).

**Pull request merged**